README.md

```diff
@@ -0,0 +1,27 @@
+# Simple Angular 2 app written in TS
+
+## Use latest TypeScript compiler
+TypeScript 1.5 beta will include everything you need. Until that is released,
+build your own TypeScript at HEAD. Follow the
+[instructions on the TypeScript repo](https://github.com/Microsoft/TypeScript/#building)
+
+## Start up the compiler
+Because we built our own TypeScript, the command to run it is longer:
+
+    $ cd ng2-ts-hello
+    $ node ../TypeScript/built/local/tsc.js -w
```

Comment:
Makes sense, didn't know about tsconfig.json, thought it is Atom-typescript specific.